### PR TITLE
1st Game Rules

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Added game rules to disable spawning of the Scourge, Legion and the Black Empire. If these rules are active, "Exodus of the Horde" and "Founding of Theramore" event chains happen after 605 year. If the Legion is disabled, Ner'zhul starts without the Legion control.

Secret changelog:
- "Terror of the Tides" event AKA naga resurfacing happens after 603 year.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2966,21 +2966,21 @@ EVTDESC_WCWAR_708_murloc;They appeared from beneath the waves. In a matter of ho
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+RULE_SCOURGE_INVASION;�RWarcraft�!: Scourge Invasion;;;;;;;;;;;;;x
+RULE_SCOURGE_INVASION_ON;On;;;;;;;;;;;;;x
+RULE_SCOURGE_INVASION_ON_DESC;Ways to create the Scourge are enabled.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]: To reduce awkwardness:
```The creation of the Scourge is allowed.```
Same with:
```The creation of the Scourge is prohibited. Pre-existing instances of the Scourge are unaffected.```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2966,21 +2966,21 @@ EVTDESC_WCWAR_708_murloc;They appeared from beneath the waves. In a matter of ho
...
+RULE_SCOURGE_INVASION_OFF;Off;;;;;;;;;;;;;x
+RULE_SCOURGE_INVASION_OFF_DESC;Ways to create the Scourge are disabled. �RThis rule doesn't affect already existing invasions.�!;;;;;;;;;;;;;x
+RULE_LEGION_INVASION;�RWarcraft�!: Legion Invasion;;;;;;;;;;;;;x
+RULE_LEGION_INVASION_ON;On;;;;;;;;;;;;;x
+RULE_LEGION_INVASION_ON_DESC;Ways to summon the Legion are enabled.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]: To reduce awkwardness:
```The summoning of the Legion is allowed.```
Same with:
```The summoning of the Legion is prohibited. Pre-existing instances of the Legion are unaffected.```

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -2966,21 +2966,21 @@ EVTDESC_WCWAR_708_murloc;They appeared from beneath the waves. In a matter of ho
...
+RULE_LEGION_INVASION_OFF;Off;;;;;;;;;;;;;x
+RULE_LEGION_INVASION_OFF_DESC;Ways to summon the Legion are disabled. �RThis rule doesn't affect already existing invasions.�!;;;;;;;;;;;;;x
+RULE_BLACK_EMPIRE_INVASION;�RWarcraft�!: Black Empire Invasion;;;;;;;;;;;;;x
+RULE_BLACK_EMPIRE_INVASION_ON;On;;;;;;;;;;;;;x
+RULE_BLACK_EMPIRE_INVASION_ON_DESC;Ways to revive the Black Empire are enabled.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]: To reduce awkwardness:
```The revival of the Black Empire is allowed.```
Same with:
```The revival of the Black Empire is prohibited. Pre-existing instances of the Black Empire are unaffected.```

**Comment by geffe1** [comment]
Seems ok, tested it.

**Pull request merged**